src/fundus/scraping/crawler.py

```diff
@@ -179,12 +185,15 @@ def crawl(
             Iterator[Article]: An iterator yielding objects of type Article.
         """
 
-        if max_articles == 0:
+        if max_articles == 0 or max_seconds == 0:
             return
 
         if max_articles is None:
             max_articles = -1
 
+        if max_seconds is None:
+            max_articles = math.inf
```

Comment:
Did you mean to overwrite `max_seconds`?
```suggestion
            max_seconds = math.inf
```

Reply:
Oops, yes, thank you. Fixed!